### PR TITLE
Update to Tomcat 9

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Comment by mogoodrich** [comment]
See: https://pihemr.atlassian.net/browse/UHM-7344

**Review by cioan — APPROVED** [review]
LGTM. Thanks @mogoodrich !

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ A few links from the OpenMRS wiki that are worth reading:
 * High-level overview of OpenMRS: https://wiki.openmrs.org/display/docs/Introduction+to+OpenMRS
 * Overview of the OpenMRS data model: https://wiki.openmrs.org/display/docs/Data+Model
 * Definition of an "OpenMRS distribution": https://wiki.openmrs.org/display/docs/OpenMRS+Distributions
+* Definition of an "OpenMRS distribution": https://wiki.openmrs.org/display/docs/OpenMRS+Distributions
```
Comment (mseaton) [inline README.md:17]: Why did you duplicate this line?

**Review by mseaton — COMMENTED** [review]
One comment.  I would feel a lot better about this if we could eliminate the "tomcat9" in favor of a set of directories that could hold up to future upgrades.  I do believe that it is possible to configure Tomcat to use directories of your choice.  Not sure how the service would be configured.  Having symbolic links created by Puppet that are then used by the debian package is also a potential option.

**Pull request merged**